docs/vlab/demo.md

```diff
@@ -86,6 +86,118 @@ graph TD
     L1 & L2 & L2 & L3 & L4 & L5 <----> S1 & S2
 ```
 
+## Utility based VPC creation
+
+### Setup VPCs
+`hhfab vlab` includes a utility to create VPCs in vlab. This utility is a `hhfab vlab` sub-command. `hhfab vlab setup-vpcs`.
+
+```console
+NAME:
+   hhfab vlab setup-vpcs - setup VPCs and VPCAttachments for all servers and configure networking on them
+
+USAGE:
+   hhfab vlab setup-vpcs [command options]
+
+OPTIONS:
+   --dns-servers value, --dns value [ --dns-servers value, --dns value ]    DNS servers for VPCs advertised by DHCP
+   --force-clenup, -f                                                       start with removing all existing VPCs and VPCAttachments (default: false)
+   --help, -h                                                               show help
+   --interface-mtu value, --mtu value                                       interface MTU for VPCs advertised by DHCP (default: 0)
+   --ipns value                                                             IPv4 namespace for VPCs (default: "default")
+   --name value, -n value                                                   name of the VM or HW to access
+   --servers-per-subnet value, --servers value                              number of servers per subnet (default: 1)
+   --subnets-per-vpc value, --subnets value                                 number of subnets per VPC (default: 1)
+   --time-servers value, --ntp value [ --time-servers value, --ntp value ]  Time servers for VPCs advertised by DHCP
+   --vlanns value                                                           VLAN namespace for VPCs (default: "default")
+   --wait-switches-ready, --wait                                            wait for switches to be ready before and after configuring VPCs and VPCAttachments (default: true)
+
+   Global options:
+
+   --brief, -b      brief output (only warn and error) (default: false) [$HHFAB_BRIEF]
+   --cache-dir DIR  use cache dir DIR for caching downloaded files (default: "/home/ubuntu/.hhfab-cache") [$HHFAB_CACHE_DIR]
+   --verbose, -v    verbose output (includes debug) (default: false) [$HHFAB_VERBOSE]
+   --workdir PATH   run as if hhfab was started in PATH instead of the current working directory (default: "/home/ubuntu") [$HHFAB_WORK_DIR]
+```
+
+### Setup Peering
+`hhfab vlab` includes a utility to create VPC peerings in VLAB. This utility is a `hhfab vlab` sub-command. `hhfab vlab setup-peerings`.
+
+```console
+NAME:
+   hhfab vlab setup-peerings - setup VPC and External Peerings per requests (remove all if empty)
+
+USAGE:
+   Setup test scenario with VPC/External Peerings by specifying requests in the format described below.
+
+   Example command:
+
+   $ hhfab vlab setup-peerings 1+2 2+4:r=border 1~as5835 2~as5835:subnets=sub1,sub2:prefixes=0.0.0.0/0,22.22.22.0/24
```

Comment:
This line has weird colours in the rendered HTML.

This is because you use `console` as the type for the code block. `console` works well for console commands with output: lines starting with prompt symbols such as `$` or `#`, like this one, get colours, and the rest is considered as output and is monochrome.

I'd recommend ditching the `console` attribute for this block (and the others). The contents are plain text anyway, not code that should get syntax highlight.

PS: Yes I know you're only moving existing blocks. Let's fix anyway :)

Reply:
agreed